update-snapshot-version.sh

```
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+JET_EE_LOCAL_PATH=$1
```

We should fail if parameter is missing